api3-whitepaper.tex

```diff
@@ -612,103 +612,66 @@ \subsection{API3 DAO}
-
-API subscription fees are commonly paid monthly or annually, as this scheme suits both API providers and their clients.
-API3 will aim to follow the same scheme for dAPIs.
-To gain access to a dAPI, the user will pay a recurring subscription fee, which may either be fixed or customized for the user based on the specific use case.
-These prices will be determined by the respective team, and will include the premium if the user wants to receive the insurance service described in Section~\ref{sec:quantifiable-security-through-insurance}.
-The payment will be able to be made in any cryptocurrency, which will be received by the DAO in API3 tokens through a liquidity pool-based decentralized exchange.
-
-API providers will be compensated periodically at fixed rates, which will fit their existing pricing models.
-This will be done using stablecoins wherever possible, yet as mentioned in Section~\ref{sec:barriers-to-api-providers-operating-oracles}, some API providers categorically reject handling cryptocurrency as payment.
-In such cases, the DAO will provide a grant that will be paid out in return of the proof that the API provider is compensated in fiat by the grantee.
-
-\subsection{Cross-platform dAPIs}
-\label{sec:cross-platform-dapis}
-
-API3 will utilize xDai~\cite{xdai:2019} as a second layer scaling solution, and thus working in a cross-platform way will be an ordinary part of its operations.
-The same workflow that will be developed to bridge xDai to Ethereum will be used to bridge to other smart contract platforms, allowing API3 to serve cross-platform dAPIs.
-These cross-platform integrations will be implemented and maintained through grants, which will be given to teams partly composed of parties from the respective smart contract platform ecosystems.
-
-We have described a scheme for compensating API providers in fiat currency in Section~\ref{sec:dapi-monetization-and-api-provider-compensation}.
-Regarding cross-platform dAPI subscription fees, we need the API3 DAO to be compensated across smart contract platforms, which is a similar problem.
-Therefore, we will use an analog of the same solution, where a grantee will pay the API3 DAO in return for being authorized to receive payments on its behalf on another smart contract platform.
-For example, if the DAO needs to be paid 100 API3 tokens as the subscription fee, the grantee will pay 90 API3 tokens to the DAO, which will result in the understanding that the grantee is authorized to receive the 100 API3 tokens-worth of payment on another chain.
-This process will be formalized as required.
-
-By implementing the data bridge and the payment channel, API3 will be able to serve other smart contract platforms without requiring them to interact with Ethereum or handle API3 tokens.
-Note that Ethereum will be used as the dispute resolution layer for the insurance service described in Section~\ref{sec:quantifiable-security-through-insurance} until a cross-platform alternative emerges.
```

Comment:
This foresees bridging of data between chains, which we moved away from as early as 2020. We are working on a mechanism for cross-chain monetization, so the bridging of payments is also irrelevant. Again, this can be revived after it gets implemented.